src/tools/ui.ts

```diff
@@ -289,6 +290,8 @@ export class UITool extends BasicTool {
         .filter((e) => e);
       elem.append(...subElements);
     }
+    if (props.enableElementDOMLog || this._basicOptions.ui.enableElementDOMLog)
```

Comment:
See L180. When props.enableElementDOMLog is false, this._basicOptions.ui.enableElementDOMLog should be ignored.